ts/main.ts

```diff
@@ -1 +1,55 @@
 /* global data */
+
+interface FormElements extends HTMLFormControlsCollection {
+  photoUrl: HTMLInputElement;
+  title: HTMLInputElement;
+  notes: HTMLTextAreaElement;
+}
+
+const $image = document.getElementsByTagName('img');
+const imf = $image[0] as HTMLImageElement;
```

Comment:
Why do the extra work of querying for all of the image elements and then finding the first one rather than just adding a unique id or class to the image element you want and targeting it that way?